### PR TITLE
add optimize argument to einsum

### PR DESCRIPTION
**Pull request by mganahl**

### CODE REVIEW

File: tensornetwork/backends/jax/jax_backend.py [hunk 1]
```diff
@@ -402,3 +402,12 @@ def expm(self, matrix: Tensor):
 
   def jit(self, fun: Callable, *args: List, **kwargs: dict) -> Callable:
     return libjax.jit(fun, *args, **kwargs)
+
+  def sum(self, tensor: Tensor, axis: Optional[Sequence[int]] = None) -> Tensor:
```
Comment (Thenerdstation) [inline tensornetwork/backends/jax/jax_backend.py:406]: Do these methods exist in the BaseBackend?

Comment (mganahl) [reply tensornetwork/backends/jax/jax_backend.py:406]: give me one sec, this was accidentially added, fixing it right now

Comment (mganahl) [reply tensornetwork/backends/jax/jax_backend.py]: OK, should be good to go!

**Comment by Thenerdstation** [comment]
Resolve conflicts and then this can be merged.

**Comment by Thenerdstation** [comment]
Bump.

**Comment by mganahl** [comment]
On it!

**Pull request merged**